framework/src/base/MooseVariableScalar.C

```diff
@@ -23,6 +23,9 @@
 #include "libmesh/numeric_vector.h"
 #include "libmesh/dof_map.h"
 
+// C++
```

Comment:
Well, originally the code looked like this:
```
// libMesh
#include "numeric_vector.h"
#include "dof_map.h"
```
Then, you guys in libmesh added "libmesh/", see `git show ff8a0bdc` for the change. At that point we should have dropped `// libMesh` comments, but they stayed. I really do think they are cpt. Obvious.  With the same logic, I do not think we need `// C++`